File: Python Word examples/tables/delete_table.py
import spirecloudword
from spirecloudword.api.tables_api import TablesApi
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.tables_api.TablesApi(configuration)

filename = "Template.docx"
index = 0
node_path = "sections/0"
inputFolder = "input"
destFilePath = "output/Tables/delete_table.docx"
result = api.delete_table(filename, node_path, index, folder=inputFolder, dest_file_path=destFilePath)

File: PHP Word Examples/TablesApiDemo/Table/deleteTable.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\TablesApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new TablesApi($configuration);

$fileName="DeleteTable_1_Temp2.docx";
$node_path1 = 'Section/0/Body/0';
$index = 1;
$dest_file_path = "Tables/DeleteTable_1.docx";
$folder = 'input';
$storage = null;
$password = null;
$apiInstance->deleteTable($fileName,$node_path,$index, $dest_file_path,$folder,$storage, $password);